test/ConveyorBelt.Tooling.Test/Parsing/SitecoreLogParserShould.cs

```diff
@@ -67,7 +67,7 @@ public void ParseCompleteErrorLevel()
             }
         }
 
-        [Fact]
+        [Fact(Skip ="Not used anymore")]
```

Comment:
why skip vs remove from code?

Reply:
Create an issue to get rid of SiteCore. This has nothing to do with the changes I made.

Reply:
Created.